.github/workflows/tests.yml

```diff
@@ -156,7 +156,7 @@ jobs:
           name: test-results-${{ github.sha }}-${{ runner.os }}-${{ matrix.default-channel }}-${{ matrix.python-version }}-${{ matrix.conda-subdir }}-${{ matrix.test-type }}-${{ matrix.test-group }}
           path: |
             .coverage
-            .test_durations_${OS}
+            tests\durations\${OS}.json
```

Comment:
Do we have to use \ here or does GHA normalize path separators for this kind of thing

Reply:
Not sure, it's probably safer to just use backslashes, despite the eye sore

Reply:
Something about YAML complicated "when is \ an escape character" parsing